### PR TITLE
[IMP] web_export_view public info

### PR DESCRIPTION
**Pull request by eLBati**

### CODE REVIEW

File: web_export_view/README.rst [hunk 1]
```diff
@@ -0,0 +1,60 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License
+
+Export Current View
+===================
+
+One of the best OpenERP’s features is exporting custom data to CSV/XLS. You can
```
Comment (pedrobaeza) [inline web_export_view/README.rst]: s/OpenERP/Odoo

File: web_export_view/README.rst [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+A lot of customers want simply to export the tree view they are looking to.
+
+If you miss this feature as us, probably you’ll find an answer into our
+web_export_view module.
+
+
+Usage
+=====
+
+After you installed it, you’ll find an additional link ‘Export current view’
```
Comment (pedrobaeza) [inline web_export_view/README.rst]: s/you‘ll/you'll

**Comment by eLBati** [comment]
Thanks @pedrobaeza , fixed

**Comment by tafaRU** [comment]
:+1:

**Comment by pedrobaeza** [comment]
The icon seems a bit large for the purpose. I used to put 128x128 px. It also seems clipped. Can you check it?

**Comment by eLBati** [comment]
@pedrobaeza resized.
But I can't see it clipped

**Comment by pedrobaeza** [comment]
OK, it's because the corner is rounded.

:+1:

**Pull request merged**